scaler/main.go

```diff
@@ -184,7 +186,7 @@ func startAdminServer(
 	})
 	mux.HandleFunc("/queue", func(w http.ResponseWriter, r *http.Request) {
 		lggr = lggr.WithName("route.counts")
-		cts := pinger.counts()
+		cts := mergeCountsWithRoutingTable(pinger.counts(), routingTable)
```

Comment:
this endpoint only needs to return the current state of the queue, not the state of the queue merged with the routing table. I think we can keep it as-is.

Reply:
does this apply to the /queue_ping as well?

Reply:
>does this apply to the "/queue_ping" as well?

yes

>for now I removed it from there as well

thank you :)